website/content/docs/k8s/k8s-cli.mdx

```diff
@@ -2,73 +2,76 @@
-| `-verbose`, `-v`                                                                                                                              | Boolean value that specifies whether to output verbose logs from the install command with the status of resources being installed.                                                                                                                                                                                                                                                                     | `false`                                |  Optional |
-| `--help`                                                                                                                                      | Prints usage information for this option.                                                                                                                                                                                                                                                                                                                                                         | none                                    | Optional |
+| Flag                          | Description                                                                                                                                                                                                                                                                                                                                                                                       | Default                                 |
+| ----------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------|
+| <nobr>`-auto-approve`</nobr>  | Boolean value that enables you to skip the installation confirmation prompt.                                                                                                                                                                                                                                                                                                                      | `false`                                 |
```

Comment:
@trujillo-adam 

this is an interesting html tag that we should consider adopting in our docs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr

Reply:
Though, maybe I should replace it with `<span style="white-space: nowrap;">`.

Reply:
That is awesome! I didn't know about the `<nobr>` tag. The other thing I started using was wrapping a `<div style="">` around curly braces to set the width of a column, but I think this tag is better.